integrations/symfony-full-configuration.rst

```diff
@@ -0,0 +1,75 @@
+Full configuration
+==================
+
+This page shows an example of all configuration values provided by the bundle.
+
+.. code-block:: yaml
```

Comment:
it would be great to eventually write this as full reference for the configuration, see http://symfony.com/doc/master/cmf/bundles/core/configuration.html for an example

(but lets merge what we have first)